### DISCUSSION TITLE
DNS over TLS or HTTPS

### DISCUSSION THREAD

**Discussion by wader-finds** [pia-foss/mobile-ios#165] [Ideas]
I recon it would be really nice if the PIA client supported using DNS over TLS or DNS over HTTPS as custom DNS options. DNS over TLS would be fairly easy I believe, its just a wrapping of the DNS protocol.

Currently I just use the "Use existing DNS" option on desktop but that isn't a thing on mobile. I have a DNS profile installed on my Iphone and it works fine when the VPN is off, but when I turn it on it starts using PIA's servers (I assume) instead of the custom ones.

I suppose it might also be possible to add a "Use existing DNS" option on mobile but I am not familiar with the APIs there.

**Comment by Lawan012** [comment]
Hey
